### DISCUSSION TITLE
better documentation for NormalizeIntensity

### DISCUSSION THREAD

**Discussion by jxchen01** [Project-MONAI/MONAI#4463] [General]
Currently, the documentation for NormalizeIntensity is not very clear, or at least not straightforward to understand. I think at least it is important to add the formular to be used for the calculation.

**Comment by Nic-Ma** [comment]
@jxchen01 ,

Thanks for your feedback, submitted a PR including it: https://github.com/Project-MONAI/MONAI/pull/4464.
